doc/release/upcoming_changes/25812.c_api.rst

```diff
@@ -0,0 +1,5 @@
+* The ``->f`` slot has been removed from ``PyArray_Descr``.
```

Comment:
You added the slot back.

Reply:
I renamed it, and will remove it completely in a follow up, thought I would add the note here already.